Wiki Reorg: Add NFT Projects (#4867)
w3f/polkadot-wiki: docs/learn/learn-nft-projects.md

---
id: learn-nft-projects
title: NFT projects on Polkadot and Kusama
sidebar_label: NFT Projects

> description: An explanation of the NFT projects in the ecosystem.

Reviewer: ```suggestion
description: Introductions to the NFT projects in the ecosystem.
```

keywords: [NFT, non-fungible token, NFT 2.0]
slug: ../learn-nft-projects
---

> 

Reviewer: Could you add the list of the projects here, so that anyone who visits the page is able to see the full list right away and then click on the project they are interested in.

```suggestion
## List of NFT Projects

- [Astar](#Astar)

```
